benchmarks/cpp/nvfuser/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ if(USE_CUDA)
   add_executable(nvfuser_bench
     batch_norm.cpp
     batch_norm_backward.cpp
+    batch_norm_nhwc.cpp
```

Comment:
Nit: I think renaming the files from `batch_norm_nhwc` to `batch_norm_channels_last` makes things clearer.
Ditto: from `batch_norm` to `batch_norm_contiguous` or `batch_norm_channels_first`